### DISCUSSION TITLE
Cannot login to server on new clients

### DISCUSSION THREAD

**Discussion by Checole** [dani-garcia/vaultwarden#3275] [General]
I have installed vaultwarden, formerly bitwarden as docker on Synology.
I downloaded the client and installed extensions on all Webbrowsers, including safari, chrome, edge and opera.

So far everything worked always fine.

Now,
first, I could not connect anymore on opera,
next I could not connect anymore on my new MacBook.

The error message is: ‘cannot read properties of null (reading iterations)’

On my old device (apart from Opera) it is still working.

Any help much appreciated!!!

**Comment by BlackDex** [comment]
See: #3082 

Make sure you are using the latest version of Vaultwarden.

**Comment by NKITHH** [comment]
Hello, 

i have the same error. 

I think i have the newest Version on my Synology (Version 2.25.1 ). 

On my old MacBook Pro Intel (Monterrey) the App and Firefox Estension are still working, also on my Android Phone. 

On my new MacBook Pro M2 (Ventura) the App and Firefox Extension give me the Error "bitwarden cannot read properties of null reading iterations". 

I tried the app (new installation) on Windows 10, it is the same error, so it seems that macOS is not the problem. 

It is like all new Logins will be failing. 

Any ideas? 

Best regards
<img width="427" alt="Bildschirmfoto 2023-02-23 um 20 54 08" src="https://user-images.githubusercontent.com/126199856/221015808-ca461e8b-9f87-472d-9c06-dac2e1818bfb.png">

**Reply by BlackDex** [reply]
Same comment for you as my previous one. Update to the latest version!

**Reply by helio58** [reply]
I have the same problem running 1.27.0
Sorry I had the wrong version.
1.27.0 works great Thanks 👍

**Comment by Checole** [comment]
BlackDex: THANK YOU!!!

Deleting the container and reinstalling solved the problem. Simple fix of a strange behaviour.

Seems I hit a nerve with my post and BlackDex advice serves for many more people.

**Comment by Taribius** [comment]
Good to know: 
The Docker Container has changed from bitwardenrs/server to vaultwarden/server. 
With this image you get the newst version and the error is fixed.
